Automattic/pocket-casts-ios
# Migrate WatchOS project to single target

Fixes #

This PR updates the project configuration to WatchOS to use a new target.

It follows the instructions on this [tech note from Apple](https://developer.apple.com/documentation/technotes/tn3157-updating-your-watchos-project-for-swiftui-and-widgetkit).

At the moment I'm just updating the project to have a single target and make the project compile and work. I'm not moving files to a single folder or remove some files that are no longer used. That will be done on a follow up PR.

When running this update on the simulator I also [hit this issue on Xcode 15.2](https://forums.developer.apple.com/forums/thread/744174) that looks it's [sorted on Xcode 15.3](https://developer.apple.com/documentation/xcode-release-notes/xcode-15_3-release-notes)


## To test

- Start the main app on a simulator device
- Stop the app
- Switch the scheme to `Pocket Casts Watch App`
- Start the watch app using an Watch OS device ( do not use the WatchOs paired option see the bug above for Xcode 15.2)
- Smoke test the app on Watch OS to see if all is working correctly

## Checklist
